src/main/kotlin/uk/gov/dluhc/notificationsapi/service/StatisticsUpdateService.kt

```diff
@@ -2,20 +2,41 @@ package uk.gov.dluhc.notificationsapi.service
 
 import org.springframework.stereotype.Service
 import uk.gov.dluhc.messagingsupport.MessageQueue
-import uk.gov.dluhc.votercardapplicationsapi.messaging.models.UpdateStatisticsMessage
+import uk.gov.dluhc.notificationsapi.dto.SourceType
 import java.util.UUID
+import uk.gov.dluhc.postalapplicationsapi.messaging.models.UpdateStatisticsMessage as PostalUpdateStatisticsMessage
+import uk.gov.dluhc.votercardapplicationsapi.messaging.models.UpdateStatisticsMessage as VoterCardUpdateStatisticsMessage
```

Comment:
This duplication feels a bit sad when they're inherently the same shape (other than the applicationId name which could be changed relatively easily). But referencing 'votercardapplicationsapi' for Postal or vice versa feels wrong

Could potentially have a whole separate yaml file for generating a service agnostic UpdateStatisticsMessage, but don't know if the duplication is enough of a problem to make that worthwhile?

Reply:
More trouble than it's worth, I think, and it doesn't really fit with the current model of these specs being "owned" by the consuming application - fine to leave as is.